### PR TITLE
Fix typos in the doc

### PR DESCRIPTION
**Pull request by GabrielCW**
Fix some typos and make the writing of Behat and Behatch more consistent in doc\index.rst

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -2,22 +2,22 @@ Behatch contexts
 ================
 
 .. image:: https://travis-ci.org/Behatch/contexts.svg?branch=master
-    :target: https://travis-ci.org/Behatch/contexts
```
Comment (sanpii) [inline doc/index.rst]: You have broken the images: https://github.com/GabrielCW/contexts/blob/fix-typos/doc/index.rst

**Comment by GabrielCW** [comment]
@sanpii broken images should be fixed now

**Comment by sanpii** [comment]
> broken images should be fixed now

Almost: you forget the last image.

And the short description ("Behatch contexts provide most common Behat tests.") disappear.

**Comment by GabrielCW** [comment]
I wasn't watching the right branch in Github, I think it should finally be right

**Comment by sanpii** [comment]
> I wasn't watching the right branch in Github, I think it should finally be right

Nice, thank you.

**Pull request merged**